CONTRIBUTING.md

```diff
@@ -73,6 +73,16 @@ Don't:
 If no reference doc exists yet, then you can use the `description` as a stopgap for explaining
 both why the audit is important and how to fix it.
 
+## Updating sample artifacts and LHR JSON
+
+```
+yarn run update:sample-artifacts # update all artifacts
+yarn run update:sample-artifacts ScriptElements # update just one artifact
+yarn run update:sample-json # update sample LHR based on sample artifacts
+```
+
+Usually you'll need to revert changes to the `*.devtoolslog.json` and `*.trace.json` files and manually review changes to `artifacts.json` to make sure they are related to your work.
```

Comment:
maybe it's unnecessary now? Or 
```suggestion
If you do a full artifacts update, usually you'll need to revert changes to the `*.devtoolslog.json` and `*.trace.json` files and manually review changes to `artifacts.json` to make sure they are related to your work.
```
I don't have a strong opinion on keeping/wording, what do you think would have been helpful for you?

Reply:
You have to revert those files even if you just do just a partial update, because we just revert the artifacts file.

> what do you think would have been helpful for you?

Just having some docs, and knowing that having lots of changes you need to revert is normal.

Reply:
> You have to revert those files even if you just do just a partial update, because we just revert the artifacts file.

that's one nice side effect of using `loadArtifacts`/`saveArtifacts`, they [automatically handle collecting and resaving](https://github.com/GoogleChrome/lighthouse/blob/03e6566fc999befc3e12ca19a3f1bc4b87e8f2d7/lighthouse-core/lib/asset-saver.js#L51-L66) the devtoolsLog and trace files too :)